Prometheus/ChildBase.cs

```diff
@@ -80,32 +83,5 @@ internal Task CollectAndSerializeAsync(IMetricsSerializer serializer, Cancellati
 
         // Same as above, just only called if we really need to serialize this metric (if publish is true).
         private protected abstract Task CollectAndSerializeImplAsync(IMetricsSerializer serializer, CancellationToken cancel);
-
-        /// <summary>
-        /// Creates a metric identifier, with an optional name postfix and an optional extra label to append to the end.
-        /// familyname_postfix{labelkey1="labelvalue1",labelkey2="labelvalue2"}
-        /// </summary>
-        protected byte[] CreateIdentifier(string? postfix = null, string? extraLabelName = null, string? extraLabelValue = null)
-        {
-            var fullName = postfix != null ? $"{_parent.Name}_{postfix}" : _parent.Name;
-
-            var labels = FlattenedLabels;
-
-            if (extraLabelName != null && extraLabelValue != null)
-            {
-                var extraLabelNames = StringSequence.From(extraLabelName);
-                var extraLabelValues = StringSequence.From(extraLabelValue);
-
-                var extraLabels = LabelSequence.From(extraLabelNames, extraLabelValues);
-
-                // Extra labels go to the end (i.e. they are deepest to inherit from).
```

Comment:
This is likely a valuable comment to preserve in the logic that was moved to TextSerializer, to emphasize that the labels are in a specific order intentionally